pkg/network/cache/BUILD.bazel

```diff
@@ -12,6 +12,7 @@ go_library(
         "//pkg/util:go_default_library",
         "//pkg/virt-launcher/virtwrap/api:go_default_library",
         "//staging/src/kubevirt.io/client-go/api/v1:go_default_library",
+        "//vendor/github.com/vishvananda/netlink:go_default_library",
```

Comment:
This and some others appear in the last commit (of the error package). I guess it leaked from prev commits, right?